articles/install-guide/setup/vs-editors/toc.yml

```
@@ -0,0 +1,4 @@
+- name: VS Code
```

Given that we have the space to do so here, I'd suggest spelling out "VS" as "Visual Studio":
```suggestion
- name: Visual Studio Code
```